package/MDAnalysis/analysis/align.py

```diff
@@ -504,6 +504,14 @@ def alignto(mobile, reference, select=None, weights=None,
     if subselection is None:
         # mobile_atoms is Universe
         mobile_atoms = mobile.universe.atoms
+        if mobile.universe == reference.universe:
```

Comment:
@jbarnoud can you double check that this is the behaviour we were aiming for?

Reply:
Looks like it is the intended behaviour :+1: